### PR TITLE
[docs] Improve the AIR introductory page

### PR DESCRIPTION
**Pull request by ericl**
## Why are these changes needed?

New sections:

air1

air2

### CODE REVIEW

File: doc/source/ray-air/getting-started.rst [hunk 1]
```diff
@@ -3,37 +3,46 @@
...
 
-Ray AIR focuses on two functional aspects:
+.. code:: bash
 
-* It provides scalability by leveraging Ray’s distributed compute layer for ML workloads.
-* It is designed to interoperate with other systems for storage and metadata needs.
+    pip install -U ray[air]
 
-Ray AIR consists of five key components:
+Why Ray AIR?
```
Comment (richardliaw) [inline doc/source/ray-air/getting-started.rst:18]: IMO this feels really overwhelming to have all this presented to me at once.

File: doc/source/ray-air/getting-started.rst [hunk 2]
```diff
@@ -3,37 +3,46 @@
 Ray AI Runtime (AIR)
 ====================
 
-Ray AI Runtime (AIR) is an open-source toolkit for building end-to-end ML applications. By leveraging Ray, its distributed compute capabilities, and its library ecosystem, Ray AIR brings scalability and programmability to ML platforms.
+Ray AI Runtime (AIR) is a scalable and unified toolkit for ML applications. AIR enables easy scaling of individual workloads, end-to-end workflows, and popular ecosystem frameworks. Unlike other toolkits, AIR can scale workloads with just Python by leveraging the Ray unified compute framework.
```
Comment (richardliaw) [inline doc/source/ray-air/getting-started.rst:6]: If I'm a user trying to do batch scoring, I can't tell within the first page whether this is a tool for me... the copy is really high level.

File: doc/source/ray-air/getting-started.rst [hunk 3]
```diff
@@ -3,37 +3,46 @@
...
+ * Scale end-to-end ML applications.
+ * Build a custom ML platform for your organization.
 
 Quick Start
 -----------
 
-Below, we demonstrate how you can use the Ray libraries in a seamless flow
-between distributed frameworks (e.g., XGBoost, Pytorch, and Tensorflow): 
+Below, we demonstrate how AIR enables simple scaling of end-to-end ML workflows
+involving multiple distributed frameworks (here showing XGBoost, Pytorch, and Tensorflow):
```
Comment (richardliaw) [inline doc/source/ray-air/getting-started.rst:45]: this reads as if each workflow touched multiple frameworks (but it doesn't)

**Review by richardliaw — COMMENTED** [review]
added comments... as an enduser I just don't know what AIR does -- only when I get to the code snippets (which are really far down)

File: doc/source/ray-air/getting-started.rst (hunk 2 above)
Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:6]: the batch or online scoring is a function of when you want to use AIR. So it should come under the subheading of When to use AIR? Just as Why Ray AIR?

File: doc/source/ray-air/getting-started.rst (hunk 3 above)
Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:45]: May be rephrase is as:
Below, we demonstrate how AIR enables simple scaling of end-to-end ML workflows using one of the multiple distributed supported frameworks (e.g., XGBoost, PyTorch, and Tensorflow):

File: doc/source/ray-air/getting-started.rst (hunk 1 above)
Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:18]: What's overwhelming? The diagram or the introduction and text?

File: doc/source/ray-air/getting-started.rst (hunk 3 above)
Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:45]: **When to use AIR?**

AIR is for both data scientists and ML engineers. Consider using AIR when you want to:
 * Scale a single workload such as training, tuning, or inference
 * Scale end-to-end ML applications.
 * Build a custom ML platform for your organization.

Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:45]: let's use data scientist and ML engineers to be consistent with our slides in the Ray AIR presentation.

File: doc/source/ray-air/getting-started.rst (hunk 1 above)
Comment (dmatrix) [reply doc/source/ray-air/getting-started.rst:18]: The single python script diagram is hard to read; the fonts a bit small and code sections blurry. I know it's from our summit slides.

Comment (richardliaw) [reply doc/source/ray-air/getting-started.rst:18]: the text is

**Comment by dmatrix** [comment]
The way I look at it, this page could really be a succinct scribe of your Ray AIR Summit vocal presentation, without all the extra slides. 
 * What and Why Ray AIR 
      * current page 
      * describe the functional components (as we had in the previous page)
        * Ray AIR focuses on two functional aspects:
         * It provides scalability by leveraging Ray’s distributed compute layer for ML workloads.
        *  It is designed to interoperate with other systems for storage and metadata needs.
       * Ray AIR consists of five key components:
         * Data processing ([Ray Data](https://github.com/ray-project/ray/data/dataset.html#datasets))
         * Model Training ([Ray Train](https://github.com/ray-project/ray/train/train.html#train-docs))
         * Reinforcement Learning ([Ray RLlib](https://github.com/ray-project/ray/rllib/index.html#rllib-index))
         * Hyperparameter Tuning ([Ray Tune](https://github.com/ray-project/ray/tune/index.html#tune-main))
         * Model Serving ([Ray Serve](https://github.com/ray-project/ray/serve/index.html#rayserve)).
 * When Ray AIR 
  * DS and MLE who wish to .... 
 * How to use Ray AIR (the quick start)

**Review by ericl — COMMENTED** [review]
I've updated with two main changes:
1. The "Why AIR" section has been moved to after the getting started code. I agree that the code is short enough that diving right in might be better than adding a much of pitch text.
2. The "What is AIR" diagram has been drastically simplified to emphasize the key use cases / libraries.

I also restored a bit of the previous text, but I'm not adding too much extra text to keep things short and concise.

**Comment by dmatrix** [comment]
I like it much better than the previous one.

**Review by richardliaw — APPROVED** [review]

**Review by xwjiang2010 — APPROVED** [review]

**Review by matthewdeng — APPROVED** [review]

**Review by stephanie-wang — APPROVED** [review]

**Review by pcmoritz — APPROVED** [review]

**Review by dmatrix — APPROVED** [review]

**Pull request merged**